rpcwebsocket.go

```diff
@@ -1323,153 +1323,435 @@ out:
 			break out
 		}
 
-		var request btcjson.Request
-		err = json.Unmarshal(msg, &request)
-		if err != nil {
-			if !c.authenticated {
-				break out
-			}
+		var batchedRequest bool
 
-			jsonErr := &btcjson.RPCError{
-				Code:    btcjson.ErrRPCParse.Code,
-				Message: "Failed to parse request: " + err.Error(),
-			}
-			reply, err := createMarshalledReply(nil, nil, jsonErr)
-			if err != nil {
-				rpcsLog.Errorf("Failed to marshal parse failure "+
-					"reply: %v", err)
-				continue
-			}
-			c.SendMessage(reply, nil)
-			continue
+		// Determine request type
+		if bytes.HasPrefix(msg, batchedRequestPrefix) {
+			batchedRequest = true
 		}
 
-		// The JSON-RPC 1.0 spec defines that notifications must have their "id"
```

Comment:
It seems a lot of code on the left-side has been removed. Can you please explain this part?

Reply:
A lot of the validations been moved further down the tree since it only applied to 1.0 rpc